server/__init__.py

```diff
@@ -149,6 +158,30 @@ def create_app():
         app.config['MAPS_API_KEY'] = secret_response.payload.data.decode(
             'UTF-8')
 
+    if cfg.ADMIN:
+        secret_client = secretmanager.SecretManagerServiceClient()
+        secret_name = secret_client.secret_version_path(cfg.SECRET_PROJECT,
```

Comment:
@Fructokinase this is another resource for website, where the oauth2 client needs to be created first and stored in Secret Manager